drv/sidecar-seq-server/src/payload.rs

```diff
@@ -7,69 +7,48 @@
 /// generator.  This code was generated by "humility rencm -g" given
 /// Aardvark output generated by running Renesas configuration software.
 ///
-#[rustfmt::skip]
 pub fn idt8a3xxxx_payload<E>(
-    mut func: impl FnMut(&[u8]) -> Result<(), E>
+    mut func: impl FnMut(&[u8]) -> Result<(), E>,
 ) -> Result<(), E> {
-
     // PAGE_ADDR = 0xc000
-    func(&[
-        0xfc, 0x00, 0xc0, 0x10, 0x20,
-    ])?;
+    func(&[0xfc, 0x00, 0xc0, 0x10, 0x20])?;
 
     // PAGE_ADDR = 0x8100
-    func(&[
-        0xfc, 0x00, 0x81, 0x10, 0x20,
-    ])?;
+    func(&[0xfc, 0x00, 0x81, 0x10, 0x20])?;
 
     // PAGE_ADDR = 0xc100
-    func(&[
-        0xfc, 0x00, 0xc1, 0x10, 0x20,
-    ])?;
+    func(&[0xfc, 0x00, 0xc1, 0x10, 0x20])?;
 
     // GPIO_USER_CONTROL.GPIO_USER_CONTROL_GPIO0_TO_7_OUT = [0]
     // GPIO_USER_CONTROL.GPIO_USER_CONTROL_GPIO8_TO_15_OUT = [0]
-    func(&[
-        0x60, 0x00, 0x00,
-    ])?;
+    func(&[0x60, 0x00, 0x00])?;
 
     // STICKY_STATUS_CLEAR.IN0_TO_7_MON_STICKY_STATUS_CLEAR = [0]
     // STICKY_STATUS_CLEAR.IN8_TO_15_MON_STICKY_STATUS_CLEAR = [0]
     // STICKY_STATUS_CLEAR.DPLL_STICKY_STATUS_CLEAR = [0]
     // STICKY_STATUS_CLEAR.DPLL_SYS_STICKY_STATUS_CLEAR = [0]
     // STICKY_STATUS_CLEAR.SYS_APLL_STICKY_STATUS_CLEAR = [0]
     // STICKY_STATUS_CLEAR.ALL_STICKY_STATUS_CLEAR = [0]
-    func(&[
-        0x64, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-    ])?;
+    func(&[0x64, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00])?;
 
     // GPIO_TOD_NOTIFICATION_CLEAR.GPIO0_TO_7_CLEAR = [0]
     // GPIO_TOD_NOTIFICATION_CLEAR.GPIO8_TO_15_CLEAR = [0]
-    func(&[
-        0x6c, 0x00, 0x00,
-    ])?;
+    func(&[0x6c, 0x00, 0x00])?;
 
     // Unknown write to 0xc170
-    func(&[
-        0x70, 0x00,
-    ])?;
+    func(&[0x70, 0x00])?;
 
     // Unknown write to 0xc171
-    func(&[
-        0x71, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-    ])?;
+    func(&[0x71, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00])?;
 
     // Unknown write to 0xc180
     func(&[
-        0x80, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+        0x80, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
```

Comment:
There's lots of free real estate for `gnarle` here, but that doesn't need to happen right away.

Reply:
Yes, we should do this soon.